content/body/progress.php

```diff
@@ -55,14 +55,14 @@
 
 
         <p>
-            Progress bars show the completion status of a current task.  It may be something that is fairly static 
+            Progress bars show the completion status of a current task.  It may be fairly static 
             for the page (e.g. A page with a progress bar indicating the delivery status of a package is unlikely 
-            to be updated in real time after the page is loaded), then you won't need an aria-live region that 
-            announces the status value.  However, if it is a task that is being reported in real time (e.g. 
-            how log it will take to upload a movie file to a web server), then you will want that information 
-            updated to the user in real time as it happens.  In the latter case, the developer and UX designer should
-            think how immediate this information should be given to screen reader users, since this information cause
-            a bit of noise, and set the aria-live level appropriately. If the user is not going to be doing anything
+            to be updated in real-time after the page is loaded), then you won't need an aria-live region that 
+            announces the status value.  However, if it is a task that is being reported in real-time (e.g. 
+            how long it will take to upload a movie file to a web server), then you will want that information 
+            updated to the user in real-time as it happens.  In the latter case, the developer and UX designer should
+            think about how immediately this information should be given to screen reader users since this information causes
+            a bit of noise and sets the aria-live level appropriately. If the user is not going to be doing anything
```

Comment:
I think the original text makes more sense, since it's not the noise that is setting the aria-live level:

```suggestion
            a bit of noise, and set the aria-live level appropriately. If the user is not going to be doing anything
```